src/core/variables/ColorVariable.ts

```diff
@@ -71,6 +71,16 @@ export class ColorVariable extends Variable implements IColorVariableParams {
   }
 
   /**
+   * Updates the selected value.
+   * @override
+   * @param {any} value The selected value.
+   */
+  updateValue(value: any): void {
```

Comment:
Including an issue tracker with further info regarding this problem:
https://github.com/Microsoft/TypeScript/issues/338